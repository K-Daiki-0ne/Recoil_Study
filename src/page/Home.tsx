import React from 'react';
import { TodoList } from '../components/TodoList/TodoList';
import { Container } from '@material-ui/core';

export function Home() {
  return (
    <Container
      maxWidth='xs'
      fixed
    >
      <TodoList />
    </Container>
  );
}