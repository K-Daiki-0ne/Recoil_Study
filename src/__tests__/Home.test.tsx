import React from 'react';
import { Home } from '../page/Home';
import Enzyme, { shallow } from 'enzyme';
import ReactSixteenAdapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new ReactSixteenAdapter() });

describe(('Render test'), () => {
  test('rendering test for Home.tsx', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  })
})