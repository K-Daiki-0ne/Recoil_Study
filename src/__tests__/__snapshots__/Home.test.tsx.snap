// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render test rendering test for Home.tsx 1`] = `
<RecoilRoot>
  <WithStyles(ForwardRef(Container))
    fixed={true}
    maxWidth="xs"
  >
    <TodoList />
  </WithStyles(ForwardRef(Container))>
</RecoilRoot>
`;
